src/controller/Component.mjs

```diff
@@ -69,10 +69,22 @@ class Component extends Base {
      * @param {String} handlerName
      * @returns {Neo.controller.Component|null}
      */
-    getHandlerScope(handlerName) {
+    getHandlerScope(handlerName, component) {
         let me     = this,
             parent = me.parent;
 
+        if (component) {
```

Comment:
not sure about this one: if a button has a string based handler, but no top level view controller, it would not get mapped.

Reply:
Yes, it would. It only bypasses looking for controllers ig the callback name is not found in the component. It will all still work as before with a Controller.

We just *added* the ability to have much simpler code. Just configure handlers into your top level view and they can be resolved there.